inc/Integrations/Airtable/AirtableIntegration.php

```diff
@@ -93,14 +94,47 @@ public static function get_query( AirtableDataSource $data_source, array $table
 		];
 
 		$output_schema = [
-			'is_collection' => false,
+			'is_collection' => true,
+			'path' => '$.records[*]',
 			'type' => self::get_airtable_output_schema_mappings( $table ),
 		];
 
 		return HttpQuery::from_array( [
 			'data_source' => $data_source,
 			'endpoint' => function ( array $input_variables ) use ( $data_source, $table ): string {
-				return $data_source->get_endpoint() . '/' . $table['id'] . '/' . $input_variables['record_id'];
+				// Normalize input variables to ensure consistent structure
+				$record_ids = [];
+				
+				// Handle both direct record_id and array of record_ids
+				if ( isset( $input_variables['record_id'] ) ) {
+					// Single record case
+					$record_ids[] = $input_variables['record_id'];
+				} else {
+					// Multiple records or array case
```

Comment:
Maybe refactor this with the other use [in the art institute block](https://github.com/Automattic/remote-data-blocks/pull/369/files#r1960311595) into a reusable function? I'm still a bit confused on what's happening here, but it seems like an operation we need multiple places.